app/uk/gov/hmrc/tariffclassificationfrontend/controllers/CaseController.scala

```diff
@@ -72,6 +72,20 @@ class CaseController @Inject()(authenticatedAction: AuthenticatedAction,
     })
   }
 
+  def attachmentsDetails(reference: String): Action[AnyContent] = authenticatedAction.async { implicit request =>
+    getCaseAndRenderView(reference, CaseDetailPage.ATTACHMENTS, c => {
+      val view = for {
+        attachments <- fileStoreService.getAttachments(c)
+        letter <- fileStoreService.getLetterOfAuthority(c)
+      } yield {
+        val filesFromApplicant = attachments.toStream.filter(_.application)
+        val filesFromClassification = attachments.toStream.filter(!_.application)
```

Comment:
toStream? We don't usually need that

Reply:
also the filer can become `_.operator.isEmpty`

Reply:
we have change  to use partition